openmetadata-ui/src/main/resources/ui/src/components/AddWebhook/AddWebhook.tsx

```diff
@@ -372,7 +393,7 @@ const AddWebhook: FunctionComponent<AddWebhookProps> = ({
                   data-testid="name"
                   id="name"
                   name="name"
-                  placeholder="name"
+                  placeholder={toLower(t('label.name'))}
```

Comment:
why we are using toLower function? instead please create a new key with lower value

Reply:
Please follow the same for wherever it is used!